explainaboard/metrics/text_to_sql.py

```diff
@@ -0,0 +1,123 @@
+"""Evaluation metrics for the text-to-sql task.
+
+Metrics contains exact set match accuracy and execution accuracy.
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import cast
+
+import numpy as np
+
+from explainaboard.metrics.metric import (
+    Metric,
+    MetricConfig,
+    MetricStats,
+    SimpleMetricStats,
+)
+from explainaboard.serialization import common_registry
+from explainaboard.third_party.text_to_sql_test_suit_eval.evaluation import (
+    evaluate as sql_evaluate,
+)
+
+
+@dataclass
+@common_registry.register("SQLExactSetMatchConfig")
+class SQLExactSetMatchConfig(MetricConfig):
+    """Configuration for SQLExactSetMatch.
+
+    Args:
+        db_dir: the path to database folder.
+        table_path: the path to table schema file.
+        etype: the evaluation type, "match" or "exec".
```

Comment:
Is this argument necessary? It seems like it should always be "match" for this class, and always be "exec" for the following class.

Reply:
It's not necessary. Let me remove it from the configs.